subsys/zbus/Kconfig

```diff
@@ -79,11 +79,17 @@ config ZBUS_ASSERT_MOCK
 
 
 config HEAP_MEM_POOL_ADD_SIZE_ZBUS
-	int
+	int "Zbus requested heap pool size."
```

Comment:
Consistent naming

```suggestion
	int "ZBus requested heap pool size."
```

Reply:
Maybe, I only commented on the changes in this PR, perhaps another PR can be opened to apply it to all?